callisto/delivery/hashers.py

```diff
@@ -0,0 +1,231 @@
+import base64
```

Comment:
removing `_load_library()` might actually not be necessary; my impulse is to stay away from doing too much magic but the more i look at it the more it seems like a solid idea. as for subclassing, it would be possible to do with the PBKDF2 hasher (and you're right, is probably a better idea) but the Argon2 hasher is only in 1.10+ so until callisto gets updated we'd have to use one locally. also, `identify_hasher()` would still need to be the version we have locally to work with existing data without encoding.